platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/PKIAuthenticationTokenFactory.java

```diff
@@ -29,17 +37,23 @@
 
   private Merlin merlin;
 
-  private String signaturePropertiesPath;
-
   /** Initializes Merlin crypto object. */
   public void init() {
-    try {
-      merlin =
-          new Merlin(
-              PropertiesLoader.loadProperties(signaturePropertiesPath),
-              PKIAuthenticationTokenFactory.class.getClassLoader(),
-              null);
-    } catch (WSSecurityException | IOException e) {
+    // NOTE: THE TRUSTSTORE SHOULD BE USED FOR CERTIFICATE VALIDATION!!!!
```

Comment:
❓  this appears to be a duplicate code block to the one above.  Can we move it into a method that both classes can use instead?